### PR TITLE
feat: docker compose driven process

### PR DESCRIPTION
**Pull request by tnunamak**

### CODE REVIEW

**Comment by tnunamak** [comment]
One outstanding task is to update the docs with instructions for submitting the staking deposit. FYI @letonchanh, in case you have any shortcuts.

**Review by volod-vana — APPROVED** [review]
Looks good

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  validator-keygen:
+    container_name: validator-setup
+    build:
+      context: https://github.com/vana-com/staking-deposit-cli.git#vana-main
+      dockerfile: Dockerfile
+    entrypoint: ["/bin/sh", "-c"]
+    # Disabled by default because it requires user confirmation
+    #- --eth1_withdrawal_address=${WITHDRAWAL_ADDRESS}
+    command: >
+      "python3 -m staking_deposit.deposit --language=${LANGUAGE:-English} new-mnemonic
```
Comment (letonchanh) [inline docker-compose.yml:179]: Without `--eth1_withdrawal_address`, the generated withdrawal credential will be `0x00` BLS address and the validators need one more step to convert it to `0x01` Eth1 address later before they are able to withdraw.

**Comment by letonchanh** [comment]
> One outstanding task is to update the docs with instructions for submitting the staking deposit. FYI @letonchanh, in case you have any shortcuts.

@tnunamak The validator can deposit with the following command:
```
cast send 0x4242424242424242424242424242424242424242 \
  "deposit(bytes,bytes,bytes,bytes32)" \
  "$PUBKEY" \ 
  "$WITHDRAWAL_CREDENTIALS" \
  "$SIGNATURE" \
  "$DATAROOT" \
  --value 35000ether \
  --private-key $PRIVATE_KEY \
  --rpc-url $RPC:80
```
`PUBKEY, WITHDRAWAL_CREDENTIALS, SIGNATURE, DATAROOT` can be found in the `deposit_data-*.json` generated by the staking deposit cli tool. `PRIVATE_KEY` is the private key of the txn sender.

I use Foundry's `cast` for simplification. Any other tool to submit tx to RPC also works.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+  #     - WITHDRAWAL_ADDRESS
+  #     - NETWORK
+  #     - LANGUAGE
+  #   profiles: ["init"]
+
+  validator-import:
+    container_name: validator-import
+    image: gcr.io/prysmaticlabs/prysm/validator:latest
+    command:
+      # TODO: this has --mainnet (Ethereum) enabled by default, is it compatible with the Vana network?
```
Comment (tnunamak) [inline docker-compose.yml:208]: @letonchanh any idea about this?

Comment (letonchanh) [reply docker-compose.yml:208]: @tnunamak We can use `--chain-config-file=/opt/vana/validator/consensus/config.yml` to distinguish ourselves from mainnet.

Comment (letonchanh) [reply docker-compose.yml:208]: I think that should be fine. This command only creates validator's wallet and it uses the validator keys generated based on the specific network we provide to the staking deposit cli tool.

**Pull request merged**